src/Akka.Persistence.Sql.Linq2Db/Journal/Types/JournalRow.cs

```diff
@@ -1,8 +1,16 @@
-﻿using LinqToDB;
-using LinqToDB.Mapping;
+﻿using System;
 
 namespace Akka.Persistence.Sql.Linq2Db.Journal.Types
 {
+    public sealed class JournalTagRow
+    {
+        public long JournalOrderingId { get; set; }
+        
+        public string TagValue { get; set; }
+        
+        public Guid WriteUuid { get; set; }
```

Comment:
No need to include `WriteUuid` on the column - OrderingId should be good enough if it's unique.